IBAnimatable/MaskType.swift

```diff
@@ -6,33 +6,33 @@
 import Foundation
 import UIKit
 
-/// Add a masking shape.
+/**
+ Mask type for masking an IBAnimatable UI element.
+ 
+ - circle: For circle shape with diameter equals to min(width, height).
+ - polygon: For polygon shape with `n` sides. (min: 3, the default: 6).
+ - star: For star shape with n points (min: 3, default:6)
+ - triangle: For isosceles triangle shape. The triangle's height is equal to the view's frame height. If the view is a square, the triangle is equilateral.
+ - wave: For wave shape with `direction` (up or down, default: up), width (default: 40) and offset (default: 0)
+ - parallelogram: For parallelogram shape with an angle (default: 60). If `angle == 90` then it is a rectangular mask. If `angle < 90` then is a left-oriented parallelogram \\-\\
+ - none: For no mask, if the input `string` of `init` is nil, the value of the neum is `.none`. If the `string` can not be mapped to supported mask type, it will set to default value `.none`.
+ */
+
```

Comment:
@lastMove duplicated comments are good for the user, but hard for us to maintain. I still want to see how Apple document their code and follow their standard. Please let me know if you find one.